youtube_dl_gui/optionsframe.py

```diff
@@ -633,11 +636,23 @@ def _set_sizer(self):
 
         vertical_sizer.AddSpacer(self.SIZE_10)
 
-        retries_sizer = wx.BoxSizer(wx.HORIZONTAL)
+        top_hor_sizer = wx.BoxSizer(wx.HORIZONTAL)
+	
+	retries_sizer = wx.BoxSizer(wx.HORIZONTAL)
         retries_sizer.Add(self.retries_text)
         retries_sizer.AddSpacer(self.SIZE_5)
         retries_sizer.Add(self.retries_spinctrl)
-        vertical_sizer.Add(retries_sizer)
+
+        maxsimdls_sizer = wx.BoxSizer(wx.HORIZONTAL)
```

Comment:
You don't actually need the *retries_sizer* and *maxsimdls_sizer* sizers.
You can pack them all together in the *top_hor_sizer* which results in fewer lines of code.